src/main/java/bowling/domain/state/Running.java

```diff
@@ -0,0 +1,34 @@
+package bowling.domain.state;
+
+import bowling.domain.pin.Pin;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Running extends State {
+    private final Pin downPins;
+
+    private Running(final Pin downPins) {
+        this.downPins = downPins;
+    }
+
+    public static Running init(final Pin fallenPins) {
+        return new Running(fallenPins);
+    }
+
+    @Override
+    protected State processDownPins(final Pin downPins) {
```

Comment:
인자로 들어오는 downPins는 **다음**에 쓰러진 핀 갯수니까
```suggestion
    protected State processDownPins(final Pin nextDownPins) {
```
는 어떨까요???

Reply:
오..오케이